Framework/API/test/AsynchronousTest.h

```diff
@@ -11,12 +11,21 @@
 #include <Poco/NObserver.h>
 #include <Poco/Thread.h>
 
+#include <condition_variable>
+#include <mutex>
+
 using namespace Mantid::Kernel;
 using namespace Mantid::API;
 using namespace std;
```

Comment:
Okay. Let's leave it as it is but can we capitalise the names to follow the standard for static variables and still remove the `using namespace std`.